modules/docker_deployment_step/assets/buildspec-deploy.yaml

```diff
@@ -16,22 +16,11 @@ phases:
       - aws configure set profile.docker-deployment.aws_access_key_id "$(jq -r '.AccessKeyId' <<<"${STS}")"
       - aws configure set profile.docker-deployment.aws_secret_access_key "$(jq -r '.SecretAccessKey' <<<"${STS}")"
       - aws configure set profile.docker-deployment.aws_session_token "$(jq -r '.SessionToken' <<<"${STS}")"
-      - |
-        aws ecr get-login-password \
-          --profile docker-deployment \
-          --region "${AWS_DEFAULT_REGION}" \
-          | docker login --username AWS --password-stdin "${ECR_URL}"
   build:
-    commands:
-      - docker load -i docker.tar
-      - export NEW_IMAGE_TAG="${CODEBUILD_BUILD_NUMBER}-${COMMIT_ID}"
-      - docker tag container-release:local "${ECR_URL}:${NEW_IMAGE_TAG}"
-      - docker push "${ECR_URL}:${NEW_IMAGE_TAG}"
```

Comment:
why are these steps removed?

Reply:
pushed in the upload step, deployed in the deploy step

It was actually the main refactor that I was after so I can reuse the upload, but add a new deploy for creating new ECS taskdef